Makefile

```diff
@@ -66,7 +71,9 @@ OCI_PUSH_ARGS  := -o type=registry
 OCI_PUSH       := docker push
 OCI_TAG        := docker tag
 
-ifeq ($(CI), true)
+ifeq ($(CI),true)
+  # ensure buildx is set up
+	_               := $(shell ./tools/ensure-buildx-builder.sh)
```

Comment:
nit: ci stages that don't run any of the build image targets would still call this